core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -2886,5 +2886,14 @@ private CarbonCommonConstants() {
 
   public static final String CARBON_ENABLE_MULTI_VERSION_TABLE_STATUS_DEFAULT = "false";
 
+  /**
+   * This flag is set to enable the feature of moving the partition data to trash
+   * when dropped partition command is given.
+   * By default it is enabled if any performance degrade then user should disable this feature.
+   */
+  @CarbonProperty
+  public static final String CARBON_ENABLE_PARTITION_DATA_TRASH =
```

Comment:
update in document file also, also add in FAQ, because here you are saying if impacted in performance, can disable.

Reply:
@akashrn5 can i disable this feature by default ?. if user want then enable and can use it. in this case we will mention only about moving partiton data to trash in documents.